test/factories/AuditEventFactorySpec.scala

```diff
@@ -243,7 +244,7 @@ class AuditEventFactorySpec extends SpecBase {
 
           val effectiveFrom = Instant.now
           val effectiveTo   = effectiveFrom.plusSeconds(99)
-          val commodity     = Commodity("030821", "Sea urchins", effectiveFrom, Some(effectiveTo))
+          val commodity     = Commodity("030821", List("Sea urchins"), effectiveFrom, Some(effectiveTo))
```

Comment:
If you really wanted to (but its not important) you could have multiple descriptions in the list and then you are testing that the audit event gets the head like you want.